Tests/PSGetInstallModule.Tests.ps1

```diff
@@ -23,6 +23,16 @@ function SuiteSetup {
     $script:PSGetLocalAppDataPath = Get-PSGetLocalAppDataPath
     $script:TempPath = Get-TempPath
     $null = New-Item -Path $script:MyDocumentsModulesPath -ItemType Directory -ErrorAction SilentlyContinue -WarningAction SilentlyContinue
+    
```

Comment:
Although I suppose the catalog tests have their own file, so maybe it ok :)

---
In reply to: [132772590](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r132772590) [](ancestors = 132772590,132599444)